instances/widgets.treasury-factory.near/widget/pages/settings/MembersEditor.jsx

```diff
@@ -231,6 +260,11 @@ useEffect(() => {
   return () => clearTimeout(timeoutId);
 }, [username]);
 
+const submissionTimeMillis = (proposal) =>
+  Number(
```

Comment:
what does this do? I don't understand well from the fn name?

Reply:
it's a copy on n we alredy have in otther places. it converts subbmission time to milisec